src/midiEditorCommands.h

```diff
@@ -10,9 +10,16 @@
 
 const int MIDI_EDITOR_SECTION = 32060;
 
-// This should be called when playback starts, as otherwise, pending note off
+// Stops the notes currently being previewed.
+// If sendNoteOff is true, note off messages are sent immediately.
+// This is used to silence the preview.
+// If sendNoteOff is false, note off messages are queued but not yet sent.
+// This is used when stopping a note preview that is immediately followed by a new preview.
+void previewNotesOff(bool sendNoteOff=true);
+
+// This must be called when playback starts, as otherwise, pending note off
 // messages for OSARA MIDI preview might interfere with MIDI playback.
```

Comment:
Please document the return value here, as it isn't immediately obvious what it means.